camel/personas/persona.py

```diff
@@ -0,0 +1,86 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import uuid
+from typing import ClassVar, Optional, Union
+
+from pydantic import BaseModel, ConfigDict, Field, PrivateAttr
+
+from camel.prompts import PersonaHubPrompt, TextPrompt
+
+
+class Persona(BaseModel):
+    r"""A persona is a character in the society.
+
+    Attributes:
+        name (Optional[str]): Name of the persona.
+        description (Optional[str]): Description of the persona.
+        t2p_prompt (Union[TextPrompt, str]): Text to Persona Prompt.
+        p2p_prompt (Union[TextPrompt, str]): Persona to Persona Prompt.
```

Comment:
Maybe we should add a principal about naming. We should not use abbreviations since our the varible names and code in CAMEL will be used by agents. Abbreviations are ambiguous.

Reply:
added in https://github.com/camel-ai/camel/pull/1209